Unable to parse alphastar replays

Bumped into this error

```
  File "/home/kevin/sc2reader/sc2reader/factories/sc2factory.py", line 85, in load_replay
    return self.load(Replay, source, options, **new_options)
  File "/home/kevin/sc2reader/sc2reader/factories/sc2factory.py", line 143, in load
    return self._load(cls, resource, filename=filename, options=options)
  File "/home/kevin/sc2reader/sc2reader/factories/sc2factory.py", line 152, in _load
    obj = cls(resource, filename=filename, factory=self, **options)
  File "/home/kevin/sc2reader/sc2reader/resources.py", line 302, in __init__
    self.load_all_details()
  File "/home/kevin/sc2reader/sc2reader/resources.py", line 446, in load_all_details
    self.load_details()
  File "/home/kevin/sc2reader/sc2reader/resources.py", line 435, in load_details
    self.length.seconds / GAME_SPEED_FACTOR[self.expansion][self.speed]
KeyError: ''
```

on [this replay](https://lotv.spawningtool.com/53947/). I got the same error on other replays, though I'm not sure it applies to all of them. The entire replay pack is at https://deepmind.com/research/open-source/alphastar-resources

@cclauss running your suggested assertion gives me

Gives me Traceback (most recent call last):

```
  File "/usr/local/lib/python3.7/site-packages/sc2reader/resources.py", line 432, in load_details
    assert self.expansion and self.speed, "{}, {}".format(self.expansion, self.speed) 
AssertionError: LotV, 
```

The reason is due to load_attribute_events doing nothing, due to:
https://github.com/ggtracker/sc2reader/blob/7ca1594ed258daad8164b9634da8afb34671192d/sc2reader/resources.py#L366
evaluating to false.

--- a/sc2reader/resources.py
+++ b/sc2reader/resources.py
@@ -431,7 +431,8 @@ class Replay(Resource):
         self.game_length = self.length
         self.real_length = utils.Length(
             seconds=int(
-                self.length.seconds / GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
+                self.length.seconds
+                / GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
             )
         )
         self.start_time = datetime.utcfromtimestamp(

--- a/sc2reader/resources.py
+++ b/sc2reader/resources.py
@@ -431,8 +431,8 @@ class Replay(Resource):
         self.game_length = self.length
         self.real_length = utils.Length(
             seconds=self.length.seconds
-            // GAME_SPEED_FACTOR[self.expansion].get(self.speed, 0.6)
-        )  # 0.6 is "normal" speed
+            // GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1.0)
+        )
         self.start_time = datetime.utcfromtimestamp(
             self.unix_timestamp - self.real_length.seconds
         )

--- a/sc2reader/resources.py
+++ b/sc2reader/resources.py
@@ -431,8 +431,8 @@ class Replay(Resource):
         self.game_length = self.length
         self.real_length = utils.Length(
             seconds=self.length.seconds
-            // GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
-        )
+            // GAME_SPEED_FACTOR[self.expansion].get(self.speed, 0.6)
+        )  # 0.6 is "normal" speed
         self.start_time = datetime.utcfromtimestamp(
             self.unix_timestamp - self.real_length.seconds
         )

--- a/sc2reader/resources.py
+++ b/sc2reader/resources.py
@@ -431,7 +431,7 @@ class Replay(Resource):
         self.game_length = self.length
         self.real_length = utils.Length(
             seconds=int(
-                self.length.seconds / GAME_SPEED_FACTOR[self.expansion][self.speed]
+                self.length.seconds / GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
             )
         )
         self.start_time = datetime.utcfromtimestamp(

--- a/sc2reader/resources.py
+++ b/sc2reader/resources.py
@@ -430,10 +430,8 @@ class Replay(Resource):
 
         self.game_length = self.length
         self.real_length = utils.Length(
-            seconds=int(
-                self.length.seconds
-                / GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
-            )
+            seconds=self.length.seconds
+            // GAME_SPEED_FACTOR[self.expansion].get(self.speed, 1)
         )
         self.start_time = datetime.utcfromtimestamp(
             self.unix_timestamp - self.real_length.seconds

